src/app/features/login/login.component.ts

```diff
@@ -1,14 +1,42 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, EventEmitter, Output, ViewChild } from '@angular/core';
+
+import { APP_ROUTS, User } from 'src/app/app.model';
+import { faEye, faEyeSlash } from '@fortawesome/free-solid-svg-icons';
 
 @Component({
   selector: 'app-login',
   templateUrl: './login.component.html',
   styleUrls: ['./login.component.scss'],
 })
-export class LoginComponent implements OnInit {
-  constructor() {}
+export class LoginComponent {
+  @ViewChild('formLogin') formLogin: any;
+
+  hidePassword = true;
+  eyeIcon = faEye;
+  eyeSlashIcon = faEyeSlash;
+  //for development
+  model: User = {
```

Comment:
For better understanding it is better to do something like this: `userData: User = InitialUserData`.

Reply:
Where `InitialUserData `stored at the `login-model.ts`.

Reply:
added files login-model.ts, and rename all files ( app.model.ts -> app-model.ts)